src/Widget/Rating.php

```diff
@@ -27,7 +27,7 @@ public function show(RatingRepository $ratingRepository){
                 'link' => 'https://kamennye-tehnologii.blizko.ru/reviews',
                 'img'=> 'images/blizko.jpg',
                 'alt'=> 'Близко.ру',
-                'stars'=> 5.0,
+                'stars'=> 4.6,
                 'rating'=> 5.0
```

Comment:
Нужно изменить `rating`, а не `stars`.
```suggestion
                'stars'=> 5.0,
                'rating'=> 4.6
```